src/AppBundle/Controller/AmendmentController.php

```diff
@@ -36,6 +36,15 @@ public function submitAction(Request $request)
                 $manager->persist($formAmendment->getData());
                 $manager->flush();
 
+                $this
+                    ->get('session')
+                    ->getFlashBag()
+                    ->add(
```

Comment:
In Symfony 2.6 you can use the shortcut method: `$this->addFlash('success', 'foo bar baz');`